info/guide/guide-chapter-3.md

```diff
@@ -494,39 +625,48 @@ Note that all comparison operators, such as `==`, `>`, `<`, should not be split.
               val a = true
               true
           }
-          .size    
+          .size
 ```
 **Note:** The parser prohibits the separation of the `!!` operator from the value it is checking.
 
 **Exception**: If a functional chain is used inside the branches of a ternary operator, it does not need to be split with newlines.
 
-**Valid example**:  
+**Valid example**:
 ```kotlin
 if (condition) list.map { foo(it) }.filter { bar(it) } else list.drop(1)
-```  
+```
 
 **Note:** If dot qualified expression is inside condition or passed as an argument - it should be replaced with new variable.
 
-**Invalid example**: 
+**Invalid example**:
 ```kotlin
- if (node.treeParent.treeParent.findChildByType(IDENTIFIER) != null) {}
+ if (node.treeParent.treeParent?.treeParent.findChildByType(IDENTIFIER) != null) {}
 ```
- 
-**Valid example**: 
+
+**Valid example**:
 ```kotlin
         val grandIdentifier = node
             .treeParent
             .treeParent
+            ?.treeParent
             .findChildByType(IDENTIFIER)
         if (grandIdentifier != null) {}
 ```
-  
+**Second valid example**:
```

Comment:
I'd put this as "another valid example" or "yet another valid example".

Reply:
Done [dd07c31](https://github.com/saveourtool/diktat/pull/1384/commits/dd07c31fed183aa7b78c96db025ce1e7a26bb77f)